chains/solana/contracts/programs/ccip-router/src/messages.rs

```diff
@@ -1,11 +1,10 @@
-use crate::{
-    BillingTokenConfig, CcipRouterError, DestChain, ReportContext,
-    TOKENPOOL_RELEASE_OR_MINT_DISCRIMINATOR,
-};
+use crate::{BillingTokenConfig, CcipRouterError, DestChain, ReportContext};
 use anchor_lang::prelude::*;
 use ethnum::U256;
 
-use crate::{ocr3base::Ocr3Report, ToTxData, TOKENPOOL_LOCK_OR_BURN_DISCRIMINATOR};
+// TODO this file has to be broken up
```

Comment:
Public interfaces that should remain the same, for example Solana2Any and Any2Solana should still be part of this module, right? And not part of the instructions crate

Reply:
Right, anything that affects the public interface won't be versioned. That's why I have to go through this file with a finer comb and split the code there, as some interface-related structs also have methods that should be upgradeable, so I'll have to make some changes here and there to achieve that. As mentioned in the PR description, I'll do that in a separate PR, to not make this grow indefinitely